test/run.js

```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-require('bulk-require')(__dirname, ['spec/**/*.js']);
+// require('bulk-require')(__dirname, ['spec/**/*.js']);
```

Comment:
Just doing a test to see whats wrong in microsoft edge (working locally though), will remove then